### ISSUE TITLE
Central Management - Database package upgrade fails with incorrect cmdlet name when passing in database parameters

### ISSUE THREAD

**Issue by ferventcoder** [chocolatey/chocolatey-licensed-issues#161]
### What You Are Seeing?

Upgrading the chocolatey-management-database package will fail if you have an existing appsettings.json (which you should on upgrade) and you are passing database parameters to the package.

~~~
ERROR: The term ‘Install-SettingsJsonFile’ is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
The upgrade of chocolatey-management-database was NOT successful.
Error while running ‘C:\ProgramData\chocolatey\lib\chocolatey-management-database\tools\chocolateyInstall.ps1’.
 See log for details.
~~~

### What is Expected?

It should not fail on a missing cmdlet.

### How Did You Get This To Happen? (Steps to Reproduce)

* Install chocolatey management database package
* Upgrade to a newer version passing any of the database/connection string parameters.
* watch it fail.

## References
* [Internal Ticket](https://gitlab.com/chocolatey/choco-licensed-management-ui/-/issues/420)
* [Zendesk](https://chocolatey.zendesk.com/hc/requests/7131)

**Comment by ferventcoder** [comment]
## Workarounds

* Delete the appsettings.json file prior to upgrade
* Do not pass database details if they have not changed during upgrade.

**Comment by ferventcoder** [comment]
This is fixed and will be released with CCM v0.3.1.

**Comment by ferventcoder** [comment]
Also provided common error and resolution to docs at https://github.com/chocolatey/choco-wiki/commit/37481c06f46da437749565443cf4c7bded371232
